src/app/domain/navigation/webviews/UrlUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { Linking } from 'react-native'
+
+import { devlog } from '/core/tools/env'
+
+/**
+ * Check if url is not HTTP or HTTPS.
+ * @param url - The URL to check
+ * @returns boolean
+ */
+export const _isNotHttpOrHttps = (url: string): boolean => {
+  return !url.startsWith('http://') && !url.startsWith('https://')
+}
+
+/**
+ * Handle Non HTTP or HTTPS urls.
+ * Be careful, this function expects a protocol that is not HTTP or HTTPS.
+ * @param url - The URL to handle
+ */
+export const _handleNonHttpOrHttps = async (url: string): Promise<void> => {
+  try {
+    await Linking.openURL(url)
+  } catch (e: unknown) {
+    if (e instanceof Error) {
+      devlog(`Could not open url ${url}`, e)
```

Comment:
We should use `Minilog.error()` here, otherwise the error won't be logged in Sentry

Reply:
👍